docs/development/Building in Ubuntu.md

```diff
@@ -45,10 +45,10 @@ Please log out and log in to active the settings. You should now be able to flas
 
 ### Unit tests
 
-To be able to run unit tests clang version 8 is required.
+To be able to run unit tests clang version 10 is required.
 
-    $ sudo apt get install clang-8 libblocksruntime-dev
-    $ make junittest
+    $ sudo apt get install clang libblocksruntime-dev
```

Comment:
This will stop working before too long, as this will install an arbitrary version of clang - better ask for a specific version to be installed.

Reply:
Nothing wrong with the line above:
```
To be able to run unit tests clang version 10 is required.
```
I'll try to maintain when new versions arrive.

Reply:
Maybe even a separate document for clang requirements to have a single source for all platforms

Reply:
Will at first do in wiki